frontend/components/simulation/SimulationForm.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <form class="flex flex-col md:flex-row gap-4 items-end" @submit.prevent="submitForm">
+        <div class="flex flex-col md:flex-row gap-4 w-full">
+            <label class="w-full md:w-1/2">
+                <TextBlock class="font-semibold text-xs">ChaosLabs access token</TextBlock>
+                <Input v-model="inputToken" :disabled="isLoading || disabled || isStarted" />
+            </label>
+            <label class="w-full md:w-1/2">
+                <TextBlock class="font-semibold text-xs">Simulation id</TextBlock>
+                <Select
+                    :value="selectedSimulationId"
+                    :disabled="isLoading || disabled || noSimulationsAvailable || isStarted"
+                    :loading="isLoading"
+                    :options="simulations"
+                    @change="selectedSimulationId = $event"
+                />
```

Comment:
Does it also let users type their own text or only allow to select pre-defined options? (we need to allow free-text input as well)

Reply:
Also `noSimulationsAvailable` should not disable the input